node_modules/oae-core/members/js/members.js

```diff
@@ -46,7 +46,7 @@ define(['jquery', 'oae.core', 'jquery.history'], function($, oae) {
 
             var url = '/api/group/' + widgetData.context.id + '/members';
             if (query) {
-                url = '/api/search/members/'  + widgetData.context.id;
+                url = '/api/search/group-members-library/'  + widgetData.context.id;
```

Comment:
As a general rule, does the project have a policy on backwards compatibility of API changes, or breaking API changes more generally? We're getting into a situation where our production deployment depends on the stability of the APIs as much as it does the stability of the rest of the platform.